spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/batch/SimpleJobService.java

```diff
@@ -245,16 +226,10 @@ public int countJobs() {
 	@Override
 	public int stopAll() {
 		Collection<JobExecution> result = jobExecutionDao.getRunningJobExecutions();
-		Collection<String> jobNames = getJobNames();
-
 		for (JobExecution jobExecution : result) {
 			try {
-				if (jobNames.contains(jobExecution.getJobInstance().getJobName())) {
-					jobOperator.stop(jobExecution.getId());
-
-				} else {
-					throw new JobStopException(jobExecution.getId());
-				}
+				stopJobExecution(jobExecution);
```

Comment:
Since nobody else is using `stopJobExecution` I would suggest one of the following:

1. Inline the code here and remove the method
2. Make the method private

I am in favor of the former.